.github/actions/build-and-deploy-api/action.yml

```diff
@@ -0,0 +1,32 @@
+name: 'build-and-deploy-api'
+description: 'Builds API Project for Production and Deploys it to a given WA Slot'
+
+inputs:
+  slot:
+    required: true
+    description: "Slot Identifier"
+  publishProfile:
+    required: true
+    description: "Azure Web App Publisher Profile"
+outputs:
+  url:
+    description: "Backend URL"
+    value: ${{ steps.wa-deployment.outputs.webapp-url }}
+
+runs:
+  using: "composite"
+  steps:
+    - run: |
+        npx nx build api --prod
+        cd dist/apps/api
+        npm i
+        npm prune --omit=dev
```

Comment:
Why don't you directly run [`npm i --omit=dev`](https://docs.npmjs.com/cli/v8/commands/npm-install#omit) on install?

Reply:
done